lib/observer/src/etop.erl

```diff
@@ -30,6 +30,7 @@
 
 -define(change_at_runtime_config,[lines,interval,sort,accumulate]).
 
+-spec help() -> ok.
```

Comment:
Description of this module refers to types without proper references that can link to actual description.